components/sbm-recipes-spring-framework/src/main/java/org/springframework/sbm/actions/spring/xml/include/ImportSpringXmlConfigAction.java

```diff
@@ -39,25 +39,28 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
+@Getter
 public class ImportSpringXmlConfigAction extends AbstractAction {
 
-    @Setter
-    @Autowired
     @JsonIgnore
-    private Configuration freemarkerConf;
```

Comment:
This does not work. The Action is created by Jackson using no-args constructor.
Teh Spring beans get injected later, see [here](https://github.com/spring-projects-experimental/spring-boot-migrator/blob/7e4f2a1a0249eff24a42c348e9c9e3421242e195/components/sbm-core/src/main/java/org/springframework/sbm/engine/recipe/DefaultActionDeserializer.java#L28)